packages/auth-server/src/account/account.js

```diff
@@ -46,11 +46,19 @@ class Account {
 	 * @returns {Promise<import('oidc-provider').UnknownObject>}
 	 */ // eslint-disable-next-line no-unused-vars
 	async claims(use, scope, claims, rejected) {
-		return {
+		const details = {
 			sub: this.accountId, // it is essential to always return a sub claim
 			email: this.user.email
 		};
 
+		if (this.user.isLpaUser) {
+			details.lpaCode = this.user.lpaCode;
+			details.isLpaAdmin = this.user.isLpaAdmin;
+			details.lpaStatus = this.user.lpaStatus;
```

Comment:
what is lpaStatus?

Reply:
added|confirmed|removed
can't login with removed
might not be needed for anything but thought I'd just include the lpa specific user details